packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-card.vue

```diff
@@ -174,8 +214,8 @@ const interventionType = computed(() => {
 
 const dynamicInterventionUnits = computed(() => {
 	let units = '';
-	const type = props.intervention.type;
-	const appliedTo = props.intervention.appliedTo;
+	const type = props.intervention.dynamicInterventions[0].type;
```

Comment:
Is there any safety needed here to know that `dynamicInterventions` isn't empty?

Reply:
This computed will only be used when a dynamic intervention has been selected. So there should be at least 1 dynamic intervention.